src/schema/root.js

```diff
@@ -562,7 +562,36 @@ export const resolvers = {
         ).result[0],
       };
 
-      return await context.datasources.getLoader("submitOrder").load(input);
+      const submitOrderRes = await context.datasources
+        .getLoader("submitOrder")
+        .load(input);
+
+      //if the request is coming from beta.bibliotek.dk, add the order id to userData service
+      if (context.profile.agency == 190101) {
```

Comment:
context og profile er aldrig undefined?

Reply:
hmmm godt spørgsmål. Jeg har tilføjet ?. for at være safe:)